src/components/tasks/card/index.tsx

```diff
@@ -14,6 +14,7 @@ import TaskLevelEdit from './TaskTagEdit';
 import { updateTaskDetails } from '@/interfaces/taskItem.type';
```

Comment:
We are just stuffing code into this File. Can we think of breaking down the component into smaller once?

Reply:
Yes, it's a good idea to break down the component into smaller pieces if it becomes too complex. Can i create the new ticket for it ?

Reply:
Yes, we can track it in new ticket.